nginx/init.sls

```diff
@@ -14,6 +14,15 @@ nginx:
     - watch_in:
       - service: nginx
 
+/etc/nginx/conf.d/https_headers.conf:
+  file.managed:
+    - source: salt://nginx/https_headers.conf
```

Comment:
Use `salt://{{ tpldir }}/files/https_headers.conf`, and make a `files` subdir to put the config file in.

I'm not sure how well you know Salt, but we'll also want the `/etc/nginx/conf.d` state from #375 and some requisites to order things correctly/restart nginx automatically on config change. I can also take care of this if you'd like.